CONTRIBUTING.md

```diff
@@ -144,6 +144,39 @@ NOTE: Once you submitted pull request to Cactus repository, step 6 is not necess
 
 NOTE: You can refer original tutorial ['GitHub Standard Fork & Pull Request Workflow'](https://gist.github.com/Chaser324/ce0505fbed06b947d962)
 
+### Directory structure
+
+Whenever you begin to contribute your codes on Hyperledger Cactus, you should follow the directory strecture on Hyperledger Cactus.
```

Comment:
@takeutak @sfuji822 

> Whenever you begin to contribute your codes on Hyperledger Cactus

Maybe we can converge on this one. When I'm looking at the directory structure outlined, I see the structure that makes sense for someone who is planning on ***using*** Cactus on a project of theirs that has business logic to be implemented so I would put this in a different part of the documentation and re-phrase it so that it says `whenever you begin to use Hyperledger Cactus ...` and it could go either in the main README.md file or some sort of GETTING-STARTED.md file that is aimed at people who do not wish to contribute to the Cactus codebase but want to use it as-is.
I'm putting more notes explaining what I mean in detail below:

If someone is planning on ***contributing*** to Cactus (but not using it per se) then that just means them writing code, but not creating a directory structure for a deployment where they have business logic plugins to implement for example.

What I mean by ***using*** here is that someone does basically this in a terminal (in a nutshell):

```sh
cd ~/
mkdir my-cool-blockchain-interop-project
cd my-cool-blockchain-interop-project
npm install @hyperledger/cactus-cmd-api-server --save --exact
mkdir -p plugins/business-logic-plugin/
cd plugins/business-logic-plugin/
vim my-cool-business-idea-plugin.ts
# Writes code for the business logic plugin, whatever that may be

mkdir -p plugins/ledger-plugin/
cd ../plugins/ledger-plugin/
vim my-cool-ledger-plugin.ts
# Writes code for the ledger they want ...

cd ../../
vim .config.json
# Writes configuration for the API server that specifies the plugins to be used (the ones implemented above)

# Starts the API server with the configuration file above so that it uses the plugins (either starts one API server or multiple ones with one plugin each if isolation is necessary, this is a deployment specific implementation detail that they can choose one way or another depending on their specific needs here I'm just showing the single process but it could be N number of them for isolation)

./node_modules/.bin/cactusapi --config-file=.config.json
```

Reply:
@petermetz , 
I think we should have some discussion regarding how to keep separated a Cactus application (e.g. an instance of the business logic plugin) from Cactus SDK, which consists with Ledger Plugins and the support library for a business logic plugin developer.
I assumed that developer will place a directory for business logic implementation outside of the Cactus project's root directory and it refers business logic plugin directory as a dependent library.
Please see [bootstrapping process](https://github.com/hyperledger/cactus/blob/master/whitepaper/whitepaper.md#422-bootstrapping-cactus-application) for your reference.

Reply:
> I assumed that developer will place a directory for business logic implementation outside of the Cactus project's root directory

@sfuji822 Okay, we are definitely on the same page then, I meant the same thing in my above comment, just explained it in a different way.
Then in this case all I need is that we swap out the word `contribute` with `use` or `deploy` in the sentence I marked above, because that's what makes it confusing whether the application bootstrapping happens `inside` our `outside` of the cactus root directory. If we can just make that one small change I'm happy to give the thumbs up and merge right after that.